v1/src/main/java/com/google/cloud/teleport/spanner/ddl/Index.java

```diff
@@ -143,6 +150,13 @@ private void prettyPrintGsql(Appendable appendable) throws IOException {
     if (interleaveIn() != null) {
       appendable.append(", INTERLEAVE IN ").append(quoteIdentifier(interleaveIn(), dialect()));
     }
+    if (options() == null) {
+      return;
+    }
+    String optionsString = String.join(",", options());
+    if (!optionsString.isEmpty()) {
+      appendable.append(" OPTIONS (").append(optionsString).append(")");
+    }
```

Comment:
We have read more things from InfoSchemaScanner, for example, `indexColumn.asc()` and `indexColumn.stored()` Which will be exported. Have you written converse DDL generation logic here for those here for Imports?

Reply:
Yes, this is part of the DDL reconstruction logic in prettyPrint. I've added reconstruction logic for all of the new entities that I've introduced.